journals/apps/journals/models.py

```diff
@@ -309,6 +309,22 @@ class JournalDocument(AbstractDocument):
 
     admin_form_fields = Document.admin_form_fields
 
+    @property
+    def deletion_warning_message(self):
+        """
+        returns the custom deletion warning message if document is being used in journal pages.
+        """
+        warning_message = "Are you sure you want to delete this document?"
+        journal_pages_title = JournalPage.objects.filter(documents=self).values_list('title', flat=True)
```

Comment:
I think instead of filtering journal page we should use `get_usage` method of document model here.